### DISCUSSION TITLE
Hydration children mismatch on，It's too difficult to find where the problem lies when such a warning appears.

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#27757] [General]
Hydration children mismatch on，It's too difficult to find where the problem lies when such a warning appears.
![image](https://github.com/nuxt/nuxt/assets/26525599/6975a8c1-35a0-4db5-bb8a-7776a43ae154)

**Comment by TheAlexLichter** [comment]
Which Nuxt version are you on? It should give a better error message nowadays. Nevertheless, the problem seems to what the `Container` is rendering. Hard to tell without code though.

Some further help:

* https://www.youtube.com/watch?v=TYyEjN0UrfA
* https://www.youtube.com/watch?v=d4-OjlicbiY

**Reply by bbhxwl** [reply]
Well, I'm using version 3.11.2, and I'm just saying that this kind of error is difficult to troubleshoot. There is a lot of code, making it difficult to accurately locate.

**Reply by bbhxwl** [reply]
> Which Nuxt version are you on? It should give a better error message nowadays. Nevertheless, the problem seems to what the `Container` is rendering. Hard to tell without code though.
> 
> Some further help:
> 
> * https://www.youtube.com/watch?v=TYyEjN0UrfA
> * https://www.youtube.com/watch?v=d4-OjlicbiY

I have watched many of your videos recently. Do you have a blog? I enjoy reading articles and they are relatively easy to translate.

**Reply by bbhxwl** [reply]
> Which Nuxt version are you on? It should give a better error message nowadays. Nevertheless, the problem seems to what the `Container` is rendering. Hard to tell without code though.
> 
> Some further help:
> 
> * https://www.youtube.com/watch?v=TYyEjN0UrfA
> * https://www.youtube.com/watch?v=d4-OjlicbiY


![image](https://github.com/nuxt/nuxt/assets/26525599/380b4efa-3fba-4b13-aa2e-c1fd9d1cec09)
![image](https://github.com/nuxt/nuxt/assets/26525599/eef07675-5cfd-423b-afac-2dc765392cc3)
Why doesn't it work?


https://www.youtube.com/watch?v=J4E5uYz5AY8&t=849s

**Reply by TheAlexLichter** [reply]
> I have watched many of your videos recently. Do you have a blog? I enjoy reading articles and they are relatively easy to translate.

I do at https://lichter.io/ but I don't write much at the moment. Hope to do so (and move some video content over in the future).

> Why doesn't it work?

Be aware that the `.env` file won't be read at runtime, only during build and dev. When starting (e.g. `node .output/server/index.mjs` you have to pass env variables "on your own").

**Reply by bbhxwl** [reply]
> > I have watched many of your videos recently. Do you have a blog? I enjoy reading articles and they are relatively easy to translate.
> 
> I do at https://lichter.io/ but I don't write much at the moment. Hope to do so (and move some video content over in the future).
> 
> > Why doesn't it work?
> 
> Be aware that the `.env` file won't be read at runtime, only during build and dev. When starting (e.g. `node .output/server/index.mjs` you have to pass env variables "on your own").

Is there Discord or any other place besides here where Nuxt communicates?

**Reply by TheAlexLichter** [reply]
Yes, there is the [Nuxt Discord](https://discord.com/invite/nuxt)
